tests/benchmarks/README.md

```diff
@@ -5,6 +5,34 @@ We have an automated system that will run any benchmark file named "test_benchma
 
 The results are displayed [here](https://rlworkgroup.github.io/benchmarks/)
 
+# Environments
+| Observations/Actions | Discrete | Continuous |
+| --- | --- | --- |
+| On-policy & Pixel | *PIXEL_ENV_SET | \ |
+| Off-policy & Pixel | Atari1M | \ |
+| State | *STATE_ENV_SET | MuJoCo1M |
```

Comment:
I suggest a more straight forward table

```suggestion
# Environments
|Algorithm	|Observation	|Action	|Environment Set|
|---|---|---|---|
|On-Policy	|Pixel	|Discrete	|PIXEL_ENV_SET|
|Off-Policy	|Pixel	|Discrete	|Atari1M|
|All	|Non-Pixel	|Discrete	|STATE_ENV_SET|
|All	|Non-Pixel	|Continuous	|MuJoCo1M|

```

Reply:
That looks good. We should also include those for meta-RL and multitask-RL. Let's modify accordingly after we have a conclusion.

Reply:
I think it would be much clearer to create a nicely-labeled list in Markdown rather than using code fences.

Reply:
The code fences are actually a table, which I think is clearer than a list